README.md

```diff
@@ -136,12 +136,22 @@ Contact Name: ITS JPO
 Contact Information: data.itsjpo@dot.gov, (888)-888-8888
 
 # Acknowledgements
-*Required - describe how users should reference your code if they use it to build additional software, list the Digital Object Identifier for this project, and (optional) list if you have a 3rd party or any specific contributor to give credit.*
+*Required - describe how users should reference your code if they use it to build additional software, list the Digital Object Identifier for this project, list the sample citations for associated report/paper and for this source code, and (optional) list if you have a 3rd party or any specific contributor to give credit.*
+
+*Sample citation should be in the below format, with the `formatted fields` replaced with details of your source code*
+
+`author_surname_or_organization`, `first_initial`. (`year`). _`program_or_source_code_title`_ (`code_version`) [Source code]. Provided by ITS CodeHub through GitHub.com. Accessed YYYY-MM-DD from `doi_url`.
```

Comment:
Sorry, last comment. I think this citation format should be in italics, since it is an instruction to the repo owner, not an example of a filled out README template.

Reply:
No worries! I wasn't sure if I should italicize it since the source code title portion of the citation is italicized, but agree that it might be more clear if we are consistently italicizing the instructions.

Reply:
So in that case, keep it all in italics _except_ the 'program or source code title' - make that plain text

Reply:
Good point - Done!